Makefile.am

```diff
@@ -1,6 +1,6 @@
 ACLOCAL_AMFLAGS = -I m4
 AUTOMAKE_OPTIONS = foreign
-SUBDIRS = src
+SUBDIRS = src test
```

Comment:
See https://github.com/smartdevicelink/bson_c_lib/pull/22/files#r245103369

Reply:
OK, I added `--with-tests` option so that `test` subdirectory is disabled by default.